Source/DolphinProcess/Darwin/DarwinDolphinProcess.h

```diff
@@ -0,0 +1,26 @@
+#ifdef __APPLE__
+
+#pragma once
+
+#include "../IDolphinProcess.h"
+#include <mach/mach.h>
+
+namespace DolphinComm
+{
+class DarwinDolphinProcess : public IDolphinProcess
+{
+public:
+  DarwinDolphinProcess()
+  {
+  }
```

Comment:
minor nitpick, not a fan of this but if its doing this via autoformat / or other files are like this, then whatever

Reply:
> other files are like this, then whatever

this is exactually the case